compiler/rustc_target/src/spec/windows_gnullvm_base.rs

```diff
@@ -0,0 +1,50 @@
+use crate::spec::{LinkArgs, LinkerFlavor, TargetOptions};
+
+pub fn opts() -> TargetOptions {
+    let mut pre_link_args = LinkArgs::new();
+    pre_link_args.insert(
+        LinkerFlavor::Gcc,
+        vec![
+            // We cannot use `-nodefaultlibs` because compiler-rt has to be
+            // passed as a path.
+            "-nolibc".to_string(),
+            "--unwindlib=none".to_string(),
+        ],
+    );
+
+    let mut late_link_args = LinkArgs::new();
+    // Order of `late_link_args*` does not matter with LLD.
+    let mingw_libs = vec![
+        "-lmingw32".to_string(),
+        "-lmingwex".to_string(),
+        "-lmsvcrt".to_string(),
+        "-lkernel32".to_string(),
+        "-luser32".to_string(),
+    ];
+    late_link_args.insert(LinkerFlavor::Gcc, mingw_libs.clone());
+
+    TargetOptions {
+        os: "windows".to_string(),
+        env: "gnu".to_string(),
+        vendor: "pc".to_string(),
+        abi: "llvm".to_string(),
+        linker: Some("clang".to_string()),
+        dynamic_linking: true,
+        executables: true,
+        dll_prefix: String::new(),
```

Comment:
Open for discussion: this is carried over from `windows_gnu_base.rs` but C/C++ libraries usually prepend `lib` like they do when targeting Linux.

Reply:
Note that if this is changed, `std::env::consts::DLL_PREFIX` needs to be changed too.